buildpacks/nodejs-npm-install/tests/snapshots/test_native_modules_are_recompiled_even_on_cache_restore.snap

```diff
@@ -26,122 +26,7 @@ source: test_support/src/lib.rs
-  - Detected Node.js version range: `>=20.0.0 <21.0.0-0`
-  - Resolved Node.js version: `20.19.0`
-- Installing Node.js distribution
-  - Reusing Node.js 20.19.0 (<arch>)
-- Done (finished in <time_elapsed>)
-
-# Heroku Node.js npm Install
-
-- Installing node modules
-  - Using npm version `10.8.2`
-  - Restoring npm cache
-  - Configuring npm cache directory
-  - Running `npm ci "--production=false"`
-
-      npm warn config production Use `--omit=dev` instead.
-
-      > dtrace-provider@0.8.8 install
-      > node-gyp rebuild || node suppress-error.js
-
-      gyp info it worked if it ends with ok
-      gyp info using node-gyp@10.1.0
-      gyp info using node@20.19.0 | linux | x64
-      gyp info find Python using Python version 3.12.3 found at "/usr/bin/python3"
-
-      gyp http GET https://nodejs.org/download/release/v20.19.0/node-v20.19.0-headers.tar.gz
-      gyp http 200 https://nodejs.org/download/release/v20.19.0/node-v20.19.0-headers.tar.gz
-      gyp http GET https://nodejs.org/download/release/v20.19.0/SHASUMS256.txt
-      gyp http 200 https://nodejs.org/download/release/v20.19.0/SHASUMS256.txt
-      gyp info spawn /usr/bin/python3
-      gyp info spawn args [
-      gyp info spawn args '/layers/heroku_nodejs-engine/dist/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
-      gyp info spawn args 'binding.gyp',
-      gyp info spawn args '-f',
-      gyp info spawn args 'make',
-      gyp info spawn args '-I',
-      gyp info spawn args '/workspace/node_modules/dtrace-provider/build/config.gypi',
-      gyp info spawn args '-I',
-      gyp info spawn args '/layers/heroku_nodejs-engine/dist/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
-      gyp info spawn args '-I',
-      gyp info spawn args '/home/heroku/.cache/node-gyp/20.19.0/include/node/common.gypi',
-      gyp info spawn args '-Dlibrary=shared_library',
-      gyp info spawn args '-Dvisibility=default',
-      gyp info spawn args '-Dnode_root_dir=/home/heroku/.cache/node-gyp/20.19.0',
-      gyp info spawn args '-Dnode_gyp_dir=/layers/heroku_nodejs-engine/dist/lib/node_modules/npm/node_modules/node-gyp',
-      gyp info spawn args '-Dnode_lib_file=/home/heroku/.cache/node-gyp/20.19.0/<(target_arch)/node.lib',
-      gyp info spawn args '-Dmodule_root_dir=/workspace/node_modules/dtrace-provider',
-      gyp info spawn args '-Dnode_engine=v8',
-      gyp info spawn args '--depth=.',
-      gyp info spawn args '--no-parallel',
-      gyp info spawn args '--generator-output',
-      gyp info spawn args 'build',
-      gyp info spawn args '-Goutput_dir=.'
-      gyp info spawn args ]
-      gyp info spawn make
-      gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
-      make: Entering directory '/workspace/node_modules/dtrace-provider/build'
-        TOUCH Release/obj.target/DTraceProviderStub.stamp
-      make: Leaving directory '/workspace/node_modules/dtrace-provider/build'
-      gyp info ok 
+      <NODE-GYP BUILD OUTPUT> 
```

Comment:
(optional) It looks like the node-gyp filter regex is greedy so is consuming the intermediate rebuild logs too?

Reply:
Good catch. Back to the drawing board with that filter.

Reply:
Using the non-greedy form now @edmorley.